src/stan-users-guide/.gitignore

```diff
@@ -0,0 +1,44 @@
+#stan intermediate files
```

Comment:
I'm pretty sure .gitignore should only go at the top of the project.

Reply:
I see I was wrong about that.  So it's fine leaving them lower.  Thanks for adding.